.github/workflows/build.yml

```diff
@@ -0,0 +1,29 @@
+name: Unit Tests
+
+on:
+  pull_request:
+    branches:
+      - develop
+
+jobs:
+  build:
+    name: ${{ matrix.os }} / ${{ matrix.python }}
+    runs-on: ${{ matrix.os }}
```

Comment:
Should we add a similar condition here to only run on our repo and not in forks? https://github.com/aws/aws-sam-cli/blob/develop/.github/workflows/update_app_templates_hash.yml#L13

Reply:
Any reason you wouldn't want it to run in PRs against `develop` in a fork?

Reply:
I don't have strong opinions on this, but not sure how it plays in with our job concurrency capacity and if it gets used up. If this is a check for merging into our repo we don't necessarily need it running on forks, but there is a genuine case if you want to test something outside of raising a PR to this repo.